x-pack/plugins/security_solution/public/app/home/index.tsx

```diff
@@ -69,10 +70,17 @@ export const HomePage: React.FC<HomePageProps> = ({ children }) => {
   const { browserFields, indexPattern, indicesExist } = useWithSource('default', indexToAdd);
 
   return (
-    <WrappedByAutoSizer data-test-subj="wrapped-by-auto-sizer" ref={measureRef}>
-      <HeaderGlobal />
-
-      <Main data-test-subj="pageContainer">
```

Comment:
This is one part where I was not sure - do we strive for semantic markup? I removed the **main** tag because I used EUI layout components (EuiFlexGroup and EuiFlexItem), but instead I can just use EUI classes on HTML elements.